SketchUp/sketchup.rb

```diff
@@ -48,8 +48,8 @@ module Sketchup
   #     # code acting on the model
   #   end
   #
-  # @return model - active model object if successful, false if
-  #   unsuccessful
+  # @return [Sketchup::Model] active model object if successful
+  # @return [false] if unsuccessful.
```

Comment:
I checked the source, this is actually returning `nil`. I got suspicion when I saw `false`. Internally we use pointers to objects so normally when we return an object we return a wrapped Ruby object - but if we have null pointers we return `nil`.